roles/ansible_service_broker/defaults/main.yml

```diff
@@ -21,3 +21,12 @@ ansible_service_broker_image_pull_policy: Always
 ansible_service_broker_sandbox_role: edit
 ansible_service_broker_auto_escalate: false
 ansible_service_broker_local_registry_whitelist: []
+
+l_asb_default_images_dict:
+  origin: 'docker.io/ansibleplaybookbundle/origin-ansible-service-broker:latest'
+  openshift-enterprise: 'registry.access.redhat.com/openshift3/ose-ansible-service-broker:${version}'
+
+l_asb_default_images_default: "{{ l_asb_default_images_dict[openshift_deployment_type] }}"
+l_asb_image_url: "{{ oreg_url | default(l_asb_default_images_default) | regex_replace('${version}' | regex_escape, openshift_image_tag) }}"
```

Comment:
`oreg_url` does not work for asb on centos because it is `openshift/origin-${component}:${version}` and asb needs `ansibleplaybookbundle/origin-ansible-service-broker`.

Reply:
Unless we push `origin-ansible-service-broker` in [`openshift`](https://hub.docker.com/u/openshift/).

Reply:
That would be ideal.

Reply:
@jwmatthews any chance that the ASB image could be pushed to openshift/origin-ansible-service-broker or do we need to amend origin to point at ansibleplaybookbundle repo?

Reply:
@sdodson we'd need to do some integration work with origin builds to accomplish.

For downstream we are setup to use same location as openshift images.

For upstream we are not integrated with origin builds, so we've been building/pushing ourselves.  Open to exploring integration with the origin builds to accomplish this if it'd help, we could add it for next sprint,  just let me know and we can start to investigate.

Reply:
CC @fabianvf @jmontleon